### PR TITLE
12.0 mig base cron exclusion

### PR DESCRIPTION
**Pull request by GoodERPJeff**

### CODE REVIEW

File: base_cron_exclusion/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (elicoidal) [inline base_cron_exclusion/README.rst:1]: see many remarks from https://github.com/OCA/server-tools/pull/1392#pullrequestreview-164502064 that might apply here

File: base_cron_exclusion/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "author": "Eficent, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/server-tools",
+    "category": "Tools",
+    "depends": [
+        "base",
+    ],
+    "data": [
+        "views/ir_cron_view.xml",
+    ],
+    "license": "AGPL-3",
```
Comment (elicoidal) [inline base_cron_exclusion/__manifest__.py:19]: add image key

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by elicoidal** [comment]
@GoodERPJeff Please make sure to update the issue for migration with all new the PR numbers

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
The commit history is clearly wrong. https://github.com/OCA/server-tools/commits/11.0/base_cron_exclusion

**Comment by HviorForgeFlow** [comment]
@GoodERPJeff attend commets + rebase would be nice to begin functional reviewing

**Pull request closed without merging**